docs/src/intro-csharp.md

```diff
@@ -122,6 +122,26 @@ dotnet test -- NUnit.NumberOfTestWorkers=5
 pwsh bin\Debug\netX\playwright.ps1 codegen
 ```
 
+## Bundle drivers for different platforms
+
+Playwright by default does bundle only the driver for the .NET publish target runtime. If you want to bundle for additional platforms, you can
+override this behavior by using either `all`, `none` or `linux`, `win`, `osx` in your project file.
+
+```xml
+<PropertyGroup>
+  <PlaywrightPlatform>all</PlaywrightPlatform>
+</PropertyGroup>
+```
+
+or:
+
+```xml
+<PropertyGroup>
+  <PlaywrightPlatform>osx</PlaywrightPlatform>
+  <PlaywrightPlatform>linux</PlaywrightPlatform>
```

Comment:
Properties are just strings, so something like this works, [because the checks are using `Contains`](https://github.com/microsoft/playwright-dotnet/pull/1955/files#diff-e3082407b006296971300d96ce43253f11b1754c334c6b7e7b81191d0a5a37fbR11):

```suggestion
  <PlaywrightPlatform>osx;linux</PlaywrightPlatform>
```